src/components/card/z-card-dictionary/styles.css

```diff
@@ -0,0 +1,77 @@
+:host {
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+}
+
+:host > div {
+  position: relative;
+  height: 523px;
+  width: 272px;
+  -webkit-transform-style: preserve-3d;
+  transform-style: preserve-3d;
+  transition: -webkit-transform 0.5s ease-out;
```

Comment:
hai verificato se l'animazione funziona crossbrowser (edge, safari, samsung mobile)?

Reply:
La compatibilità col cellulare la scopriamo portando la roba su dev, da locale faccio fatica.. Safari lo facciamo controllare a qualcuno.

Reply:
per verificare l'animazione senza andare su dev potresti mettere lo snippet su codepen e poi puoi provare

Reply:
Flippa in ogniddove!